src/Symfony/Component/HttpClient/Response/MockResponse.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpClient\Response;
+
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+/**
+ * A test-friendly response.
+ *
+ * @author Gary PEGEOT <garypegeot@gmail.com>
+ */
+class MockResponse implements ResponseInterface
+{
+    use ResponseTrait;
+
+    protected const DEFAULT_INFO = [
```

Comment:
should be private rather than protected IMO, following our convention of not using `protected` without having a clear use case for inheritance.

Reply:
and change the access to `self::`, as using `static::` to access a private thing is broken.

Reply:
Fixed